apps/desktop/src/routes/reduxExample/+page.svelte

```diff
@@ -15,15 +20,17 @@
 	 * `location = '/reduxExample'` in the console.
 	 */
 
-	const appState = getContext(AppState);
-	const appDispatch = getContext(AppDispatch);
+	const appState = getContext(DesktopState);
+	const appDispatch = getContext(DesktopDispatch);
```

Comment:
We ought to only get the DesktopState when we actually need the desktop specific slices, so I've only updated this one component.